extensions/amp-consent/0.1/test/test-amp-consent.js

```diff
@@ -680,11 +681,52 @@ describes.realWin(
             'consentState': 2,
             'consentString': 'mystring',
             'isDirty': true,
+            'consentMetadata': undefined,
           });
         });
       });
     });
 
+    describe('consent metadata', () => {
+      let ampConsent;
+
+      beforeEach(() => {
+        const defaultConfig = dict({
+          'consents': {
+            'ABC': {
+              'checkConsentHref': 'https://response1',
+            },
+          },
+        });
+        const consentElement = createConsentElement(doc, defaultConfig);
+        doc.body.appendChild(consentElement);
+        ampConsent = new AmpConsent(consentElement);
+      });
+
+      it('should error and return undefined on invalid metadata', () => {
+        const spy = env.sandbox.stub(user(), 'error');
```

Comment:
We have a method to test errors. Please check `allowConsoleError()`

Reply:
Related to #28438